package.json

```diff
@@ -78,7 +81,10 @@
   "dependencies": {
     "@openmrs/esm-api": "^1.2.2",
     "@openmrs/esm-module-config": "^1.2.0",
+    "@openmrs/esm-root-config": "^1.2.0",
```

Comment:
We should drop this dependency. Since we started the repo the way how to publish / distribute was refined and this is no longer needed.